docs/concepts.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Concepts
+nav_order: 2
+---
+# Concepts
+{: .no_toc}
+
+## Table of Contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+---
+## Key Terms
+- **Inferno Test Kit**: A distributable set of tests and tools built and packaged
```

Comment:
Re: "What is a test kit?" -- I think it might be useful to maybe offer some narrative on the intention behind a kit. I think the description here does a great job outlining the capabilities, but two people could read this and come to very different conclusions about what a kit should look like. Maybe some guidance on: should test kits focus on specificity or breadth (i.e. is it better to have a kit that tests one thing really well or a bunch of different features but not rigorously?) Or does this distinction not matter, meaning you can say it's completely up to the writer. I also think it makes sense to distinguish kits as the packaging behind one or more suites - it's my understanding that we're not passing around suite's as gems, just kits. Pointing to the kit as the uniform 'package' and how that helps standardize pulling in different test kits and their components will help differentiate them from suites.

Reply:
This is 100% what I was hoping we could nail down a bit more.